content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -449,7 +450,11 @@ Each feature gate is designed for enabling/disabling a specific feature:
    for more details.
 - `CPUManager`: Enable container level CPU affinity support, see
   [CPU Management Policies](/docs/tasks/administer-cluster/cpu-management-policies/).
-- `CRIContainerLogRotation`: Enable container log rotation for cri container runtime.
+- `CRIContainerLogRotation`: Enable container log rotation for cri container runtime. Use kubelet flags `container-log-max-size` and `container-log-max-files`
+  to configure the maximum size for each log file and the maximum number of files allowed for each container respectively. The rotated log files can be accessed
+  by logging onto the node and reading the logs from the log directory or an external logging agent can be used to grab the rotated logs from all the nodes and
+  store them in a remote location.
+  Note: `kubectl logs` does not currently grab logs from the rotated files.
```

Comment:
it may be too much information for feature-gate page. It may be better to update this page: https://kubernetes.io/docs/concepts/cluster-administration/logging/

Reply:
Fixed